documentation/modules/ROOT/pages/transformations/partition-routing.adoc

```diff
@@ -108,28 +122,150 @@ Given this `Products` table
 Based on the configuration, the SMT routes change events for the records that have the field name `hammer` to the same partition.
 That is, the items with `id` values `104`, `105`, and `106` are routed to the same partition.
 
+// Type: concept
+// Title: Example: Advanced configuration of the {prodname} partition routing SMT
+// ModuleID: advanced-configuration-of-the-debezium-partition-routing-smt
+[[example-advanced-partition-routing-configuration-example]]
+== Example: Advanced configuration
+
+Suppose you have different data collections (t1, t2) routed to same topic (my_topic), and you want to partition events from data collection t1 using field f1
+and events from data collection t2 using field f2.
+
+This will be the configuration
+
+[source]
+----
+transforms=PartitionRouting
+transforms.PartitionRouting.type=io.debezium.transforms.partitions.PartitionRouting
+transforms.PartitionRouting.partition.payload.fields=change.f1,change.f2
+transforms.PartitionRouting.partition.topic.num=2
+transforms.PartitionRouting.predicate=myTopic
+
+predicates=myTopic
+predicates.myTopic.type=org.apache.kafka.connect.transforms.predicates.TopicNameMatches
+predicates.myTopic.pattern=my_topic
+----
+
+Note that in this configuration we omitted the part to configure the re-routes of events from different topic to a specific one. See {link-prefix}:{link-topic-routing}[Topic Routing SMT].
```

Comment:
As much as possible, it's best to avoid sentences that begin with the construction "Note that ..." or similar. Place content that should be in a note in an asciidoc admonition (Note, Important, Warning, and so forth). Otherwise, rework the content into the main flow.